### ISSUE TITLE
No "find & replace" for bad formatted XML?

### ISSUE THREAD

**Issue by Pierre-de-la-Verre** [microsoft/XmlNotepad#87]
I have a XML like this

```
<?xml version="1.0" encoding="utf-8"?>
<DataChecks>
<DataCheck ID="1" ParentDataCheckID="-1" Name="Test ---------------------------------------------------------------" Description="" DataCheckSqlStatement="" WithoutRLS="false" Active="true" ModuleName="" />
  .... 80 lines more
</DataChecks>
```
XML Notepad tells me "The DataChecks-Element is not declared", and "find" and "replace" are doing nothing: no search, no error, no report, ...
Is this intended?

**Comment by clovett** [comment]
This is what I see.  Perhaps you have a "Schema" loaded in the "View/Schemas..." dialog that is defining a schema for any XML document that uses no namespace and that is mistakenly being applied to this document.  Try deleting any schemas you see in the  "View/Schemas..." dialog and see if that helps.

![image](https://user-images.githubusercontent.com/3336415/115068420-18ef6000-9ea7-11eb-84bd-b88e3f0b2851.png)

**Comment by Pierre-de-la-Verre** [comment]
here is a screencast: https://autode.sk/3sqBaux

Maybe a user problem?

**Comment by lovettchris** [comment]
Oh, thanks for the video, it seems that "find" is broken in some cases, thanks for he video.  I'll check it out.

**Comment by lovettchris** [comment]
thanks for the great bug!! 
fixed in 2.8.0.29.
See [commit 4aeae9b1621969fd97c7d392eab2da2ec7755ab3](https://github.com/microsoft/XmlNotepad/commit/4aeae9b1621969fd97c7d392eab2da2ec7755ab3).

**Comment by Pierre-de-la-Verre** [comment]
Thanks for fixing 👍
